catalog/installWeather.sh

```diff
@@ -12,24 +12,18 @@ source "$CATALOG_HOME/util.sh"
 echo Installing Weather package.
 
 createPackage weather \
-    -a description "Services from The Weather Company" \
+    -a description "Services from IBM Weather Insights" \
     -a parameters '[ {"name":"apiKey", "required":false} ]'
 
 waitForAll
 
 install "$CATALOG_HOME/weather/forecast.js" \
     weather/forecast \
-    -a description 'Weather.com 10-day forecast' \
-    -a parameters '[ {"name":"latitude", "required":true}, {"name":"longitude", "required":true}, {"name":"apiKey", "required":true, "type":"password", "bindTime":true} ]' \
+    -a description 'IBM Weather Insights 10-day forecast' \
+    -a parameters '[ {"name":"latitude", "required":true}, {"name":"longitude", "required":true},{"name":"language", "required":true},{"name":"units", "required":true}, {"name":"apiKey", "required":true, "type":"password", "bindTime":true} ]' \
```

Comment:
some of these annotations carry the `required` attribute but the `sampleInput` does not specify them (e.g., `language` and `units`).